### ISSUE TITLE
Calcaneal apophysitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11191]
**Preferred term label:**
Calcaneal apophysitis

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0264097
Lateral foot x-ray showing increased density in the calcaneal apophysis or fragmentation of the apophysis

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sever's disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001351
- label: Calcaneal apophysitis
- parents: [Abnormality of the calcaneus (HP:0008364)](https://hpo.jax.org/app/browse/term/HP:0008364)
- pmid: PMID:38544602
- def: Inflammation of the calcaneal apophysis (bony projection where the Achilles tendon attaches to the heel). Lateral foot x-ray shows increased density in the calcaneal apophysis or fragmentation of the apophysis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11191
- synonyms:
